modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/gossip/Gossip.scala

```diff
@@ -38,7 +39,7 @@ object Gossip {
     for {
       counter <- Ref.of[F, PosLong](PosLong(1L))
       time <- Clock[F].realTime
-      generation <- PosLong.from(time.toMillis).leftMap(new RuntimeException(_)).liftTo[F]
+      generation <- PosLong.from(time.toMillis).map(Generation(_)).leftMap(new RuntimeException(_)).liftTo[F]
```

Comment:
I think that we can move the creation of a generation out of gossip unless you want to do it in the following PRs.

Reply:
I'm working on that right now in separate PR, when reusing it for the session token.